app/javascript/crayons/Icons/Icon.jsx

```diff
@@ -0,0 +1,20 @@
+import { h } from 'preact';
+import SVG from 'react-inlinesvg';
+import classNames from 'classnames/bind';
+
+export const Icon = (props) => {
```

Comment:
It took looking at the implementation details for me to know what the `native` prop was for, which got me thinking for all these new components we should add [JSDoc comments](https://jsdoc.app/about-getting-started.html) to document what all the props do

Reply:
I've actually just pushed a commit with tons of documentation including Controls descriptions... :)